### ISSUE TITLE
Inner tf coils gap size

### ISSUE THREAD

**Issue by billingsley-john** [ukaea/paramak#292]
If gap size is too large, strange geometries can be created. A check needs to be put in place to prevent this. (see image)

**Comment by billingsley-john** [comment]
![Uploading Screenshot from 2020-09-24 15-19-24.png…]()

**Comment by Shimwell** [comment]
We might be able to count the .Solids() and flag this error up

something like this
```
if len(InnerTfCoilsFlat.solid.Solids()) > self.number_of_coils:
```

**Comment by billingsley-john** [comment]
Resolved in #309
